apps/web/cache-handler.mjs

```diff
@@ -11,42 +11,38 @@ const createTimeoutPromise = (ms, rejectReason) => {
 CacheHandler.onCreation(async () => {
   let client;
 
-  if (process.env.REDIS_URL && process.env.ENTERPRISE_LICENSE_KEY) {
-    try {
-      // Create a Redis client.
-      client = createClient({
-        url: process.env.REDIS_URL,
-      });
+  try {
```

Comment:
We still support both Redis cache handler and simple in-memory cache so we should still check if REDIS_URL is present. Otherwise this will break on existing setups without Redis.